rise-google-sheet.html

```diff
@@ -237,17 +237,49 @@
       },
 
       _prepareResponse: function (resp) {
-        var response = {};
+        var response = {},
+          firstRowCols = 0,
+          totalCols = 0;
 
         /*
          Provide an empty array if values property missing in response object. This can occur if any range
          values (minColumn, maxColumn, minRow, maxRow) are out of scope of the data entered in worksheet
         */
         response.results = (resp.values) ? resp.values : [];
 
+        // Workaround for this issue - https://goo.gl/rsazDP.
+        firstRowCols = (response.results && response.results.length > 0) ? response.results[0].length : 0;
+        totalCols = this._getTotalCols(response.results);
+
+        // Append cells to the first row until it matches the total column count.
+        if (firstRowCols < totalCols) {
```

Comment:
@donnapep That will suffice for our Widget, but this component gets used elsewhere also (eg. Creative projects). I think we need to fill all empty values to ensure there are no issues for all consumers of the component.